### ISSUE TITLE
Option `--output-type pdf` causes PDF to be blank (with wrong geometry)

### ISSUE THREAD

**Issue by fnoy** [ocrmypdf/OCRmyPDF#482]
With following PDF file:
[test.pdf](https://github.com/jbarlow83/OCRmyPDF/files/4109466/test.pdf)

If you do `ocrmypdf --output-type pdf test.pdf foo.pdf`, the file foo.pdf is 10 times smaller than the original, and is blank when you open it. Remove the option `--output-type pdf` and the problem is gone.

I noted this:
```
$ pdfimages foo.pdf img
$ identify -format "%@" img-000.pbm
identify-im6.q16: geometry does not contain image `img-000.pbm' @ warning/attribute.c/GetImageBoundingBox/247.
0x0+2023+2866
```
The geometry 0x0 is obviously problematic.

```
$ ocrmypdf --version
9.4.0+dfsg
$ tesseract --version
tesseract 4.1.1
 leptonica-1.79.0
  libgif 5.1.9 : libjpeg 6b (libjpeg-turbo 1.5.2) : libpng 1.6.37 : libtiff 4.1.0 : zlib 1.2.11 : libwebp 0.6.1 : libopenjp2 2.3.1
 Found AVX2
 Found AVX
 Found FMA
 Found SSE
 Found libarchive 3.4.0 zlib/1.2.11 liblzma/5.2.4 bz2lib/1.0.8 liblz4/1.9.1 libzstd/1.4.3
```

**Comment by jbarlow83** [comment]
I couldn't reproduce.

What platform is this?

**Comment by fnoy** [comment]
Debian.
```
$ cat /etc/debian_version
bullseye/sid
```

Here is the output file I get: 
[foo.pdf](https://github.com/jbarlow83/OCRmyPDF/files/4138140/foo.pdf)

**Comment by jbarlow83** [comment]
Do you have jbig2 installed it? What version?

It seems that jbig2 conversion was attempted by somehow failed and a blank jbig2 image was inserted instead. If you disable jbig2 conversion (by uninstalling jbig2/suppressing from path) it should work. 

jbig2 is generally desirable to have however.

**Comment by fnoy** [comment]
Yes I have jbig2 installed: jbig2enc v0.28 and jbig2dec v0.17.
Thanks for the tip about disabling jbig2.

That would be nice to have at least a warning that the conversion failed with jbig2. Because if the user does the ocr of many pdf files in batch mode, there is a risk that the user ends up with blank pdfs without noticing it, which would be a serious problem.

**Comment by damnms** [comment]
facing the same issue, --output-pdf leads to nearly empty pages 
oli@lucy:~/jbig2enc/src$ ./jbig2 --version
jbig2enc 0.28

**Comment by jbarlow83** [comment]
@fnoy You have an old version of ocrmypdf.

@damnms What version of ocrmypdf do you have?

It's quite possible this issue was fixed by #732 or commit 49734d which is in v10.2.1+

**Comment by gnewell3** [comment]
Hi I can confirm this issue has been resolved.

Tested with the test pdf provided on ocrmypdf:v13.6.2 and it gives no issues. All perfect crisp OCR with text layer ;)

Believe can be closed :)!
